spec/controllers/admin/pdfs_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+require 'rack/test'
+
+RSpec.describe Admin::PdfsController, type: :controller do
+
+  it "should create an image" do
+    pdf_file = Rack::Test::UploadedFile.new('spec/fixtures/sample_pdf.pdf', 'application/pdf')
+    pdf = create(:pdf, pdf_file: pdf_file)
+
+    assert_difference('Pdf.count') do
+      post :create, pdf: { page: pdf.page,
+                          slot: pdf.slot,
+                          pdf_file: pdf_file
+                          }
+    end
+
```

Comment:
Did you try this test without using a gem? I'm curious because I used this without a gem:
expect{ delete :destroy, id: @company }.to change(Company,:count).by(-1)